lib/ovirt/disk_profiles.rb

```diff
@@ -0,0 +1,19 @@
+module OVIRT
+  class DiskProfile < BaseObject
```

Comment:
Hey friend, 


There are a couple of books that inspire me to do really great  things,  here is  the list of  that books <http://practice.goscrappin.com/lnjsdo>

Kind regards, Doug Bishop